doc/integrator/querier.rst

```diff
@@ -25,16 +25,16 @@ The web service configuration is done in the ``vars_<project>.yaml`` file:
                         separator: ","
 
 The ``dbsession: "<session name>"`` in the ``enum`` is just a shortcut
-if almost of the session are similar in the layers. If the dbsession isn't
-defined we use the main DB session.
+if almost of the session are similar in the layers. If ``dbsession`` is not
```

Comment:
we can fill the Sessions define here:
https://github.com/camptocamp/c2cgeoportal/blob/master/c2cgeoportal/models.py#L69-L71
and it's used here:
https://github.com/camptocamp/c2cgeoportal/blob/master/c2cgeoportal/views/layers.py#L397